# Let it us EasyBuild provided Boost instead of it's own version
# wpoely86@gmail.com
diff -ur KAT-Release-2.4.2.orig/lib/Makefile.am KAT-Release-2.4.2/lib/Makefile.am
--- KAT-Release-2.4.2.orig/lib/Makefile.am	2018-05-04 10:55:58.000000000 +0200
+++ KAT-Release-2.4.2/lib/Makefile.am	2018-09-01 20:23:43.207477276 +0200
@@ -6,7 +6,6 @@
 lib_LTLIBRARIES = libkat.la
 
 libkat_la_LDFLAGS = \
-	-L$(top_builddir)/deps/boost/build/lib/ \
 	-version-info 2:4:2
 
 LIBS = \
@@ -41,7 +40,6 @@
 
 libkat_la_CPPFLAGS = \
 	-I$(top_srcdir)/deps/jellyfish-2.2.0/include \
-	-I$(top_srcdir)/deps/boost/build/include \
 	-I$(top_srcdir)/lib/include \
 	@AM_CPPFLAGS@
 
Only in KAT-Release-2.4.2/lib: Makefile.am~
diff -ur KAT-Release-2.4.2.orig/src/Makefile.am KAT-Release-2.4.2/src/Makefile.am
--- KAT-Release-2.4.2.orig/src/Makefile.am	2018-05-04 10:55:58.000000000 +0200
+++ KAT-Release-2.4.2/src/Makefile.am	2018-09-01 20:24:23.826012318 +0200
@@ -14,7 +14,6 @@
 kat_CPPFLAGS = \
 	-I$(top_srcdir)/deps/seqan-library-2.0.0/include \
 	-I$(top_srcdir)/deps/jellyfish-2.2.0/include \
-	-I$(top_srcdir)/deps/boost/build/include \
 	-I$(top_srcdir)/lib/include \
 	-DKAT_SCRIPTS='"$(datadir)/kat/scripts"' \
 	-DKAT_EXECPREFIX='"$(exec_prefix)"' \
@@ -25,11 +24,11 @@
 	@AM_LDFLAGS@
 
 kat_LDADD = \
-	$(top_builddir)/deps/boost/build/lib/libboost_timer.a \
-	$(top_builddir)/deps/boost/build/lib/libboost_chrono.a \
-	$(top_builddir)/deps/boost/build/lib/libboost_filesystem.a \
-	$(top_builddir)/deps/boost/build/lib/libboost_program_options.a \
-	$(top_builddir)/deps/boost/build/lib/libboost_system.a \
+	-lboost_timer \
+	-lboost_chrono \
+	-lboost_filesystem \
+	-lboost_program_options \
+	-lboost_system \
 	$(top_builddir)/deps/jellyfish-2.2.0/.libs/libkat_jellyfish.la \
 	$(top_builddir)/lib/libkat.la \
 	@AM_LIBS@
Only in KAT-Release-2.4.2/src: Makefile.am~
diff -ur KAT-Release-2.4.2.orig/tests/Makefile.am KAT-Release-2.4.2/tests/Makefile.am
--- KAT-Release-2.4.2.orig/tests/Makefile.am	2018-05-04 10:55:58.000000000 +0200
+++ KAT-Release-2.4.2/tests/Makefile.am	2018-09-01 20:24:41.594808903 +0200
@@ -31,7 +31,6 @@
 	-I$(top_srcdir)/lib/include \
 	-I$(top_srcdir)/deps/seqan-library-2.0.0/include \
 	-I$(top_srcdir)/deps/jellyfish-2.2.0/include \
-	-I$(top_srcdir)/deps/boost/build/include \
 	-DDATADIR=\"$(srcdir)/data\" \
 	@AM_CPPFLAGS@
 
@@ -57,8 +56,6 @@
 
 check_unit_tests_LDFLAGS = \
 	-static \
-	-L$(top_builddir)/deps/boost/build/lib \
-	-Wl,-rpath $(top_builddir)/deps/boost/build/lib \
 	@AM_LDFLAGS@
 
 check_unit_tests_LDADD = \
Only in KAT-Release-2.4.2/tests: Makefile.am~
